tests/console/systemd_nspawn.pm

```diff
@@ -84,7 +86,7 @@ sub run {
         zypper_call 'in busybox-static';
         assert_script_run 'cp -v /usr/bin/busybox-static ./oci_testbundle/rootfs/bin/busybox';
     }
-    if (script_run('systemd-nspawn --oci-bundle=/tmp/oci_testbundle |& grep "Failed to resolve path rootfs"') == 0) {
+    if (script_run('(systemd-nspawn --oci-bundle=/tmp/oci_testbundle || true) |& grep "Failed to resolve path rootfs"') == 0) {
```

Comment:
sorry, I assumed you would also answer for both cases. Or is there a reason to not use `validate_script_output` here?

Reply:
Turns out that `validate_script_output` doesn't work nicely here as it expects exit code `0` and doesn't seem to allow specifying a list of allowed exit codes:

```
validate_script_output($script, $code | $regexp [, timeout => $timeout] [,quiet => $quiet])
```

Therefore the test fails:
http://kazhua.qa.suse.de/tests/245#step/systemd_nspawn/128